components/motor/gpio/basic.go

```diff
@@ -284,11 +284,6 @@ func (m *Motor) GoFor(ctx context.Context, rpm, revolutions float64, extra map[s
 		return errors.Wrap(err, "error in GoFor")
 	}
 
-	if revolutions == 0 {
```

Comment:
just to confirm, we are choosing not to error when revolutions is 0, but rahter we will return a value from goForMath that results in the motors not moving

Reply:
now we are erroring